schemas/api/iFrameRestyle.schema.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://fdc3.finos.org/schemas/next/api/iframeRestyle.schema.json",
+  "title": "iframe Restyle",
+  "description": "Message from a UI iframe to the DA proxy code (setup by `getAgent()`) with updated styling information to apply to the iframe. Can be used to implement a pop-open or close interaction or other transition needed by a UI implementation.",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "#/$defs/IframeRestyleBase"
+    },
+    {
+      "$ref": "iFrameMessage.schema.json"
+    }
+  ],
+  "$defs": {
+    "IframeRestyleBase": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "title": "iframeRestyle Message Type",
+          "const": "iframeRestyle"
+        },
+        "payload": {
+          "title": "iframeRestyle Payload",
+          "type": "object",
+          "properties": {
+            "updatedCSS": {
+              "title": "Updated CSS",
+              "type": "object",
+              "description": "A constrained set of CSS properties that should be applied to the iframe. Note `position` cannot be set, and should always be `fixed`.",
+              "properties": {
+                "height": {"type": "string", "title": "height", "description": "The updated height of the iframe"},
+                "width": {"type": "string", "title": "width", "description": "The updated width of the iframe"},
+                "zIndex": {"type": "string", "title": "zIndex", "description": "The updated zindex to apply to the iframe"},
```

Comment:
Hi @kris,

Could we change these to be css properties instead of javascript ones?

Reply:
In other words, which do you think is better, the CSS-style properties ( `z-index` ) or the javascript-style properties ( `zIndex` )? The way Rob's current implementation works assumes CSS-style properties. If we want to keep javascript-style properties, we would need to change the implementation.

Which do you think makes more sense for the spec? Rob prefers the CSS-style, and I don't have an opinion either way.

Reply:
I don't think it matters that much but as you ask:

I'm not aware of any frontend frameworks that use CSS-style properties (outside of actual CSS files) - to my knowledge, everyone uses the javascript-style CSS props as standard and as these would be applied in code it would make sense to stick to the javascript style.

Also, JSON (JavaScript Object Notation) would usually stick to its own naming conventions for property names, which gives you the Javascript style name. That's easily applied with `element.style["propertyName"] = value`. The alternative isn't much worse with the [CSSStyleDeclaration](https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration) utility fn `setProperty`, e.g.  `element.style.setProperty("property-name", value) - I've just never seen anyone bother doing this.

If you really want this changed @robmoffat let me know - I don't care either, it just seems odd (given that all of this will happen in javascript).